source/PlayState.hx

```diff
@@ -1,16 +1,187 @@
 package;
 
+import flixel.text.FlxText;
+import entities.collectibles.Coin;
+import entities.terrain.Lava;
+import entities.projectiles.Fireball;
+import flixel.FlxObject;
+import flixel.FlxG;
+import flixel.group.FlxGroup.FlxTypedGroup;
+import entities.player.Hero;
+import entities.terrain.Wall;
 import flixel.FlxState;
 
 class PlayState extends FlxState
 {
-	override public function create()
+
+	private static var WORLD_LEFT(default,never) = -(1280*2);
+	private static var WORLD_RIGHT(default,never) = 1280*2;
+	private static var WORLD_TOP(default,never) = -(960*2);
+	private static var WORLD_BOTTOM(default,never) = 960*2;
+
+	private static var WALL_COUNT(default, never) = 10;
+	private static var WALL_START_X(default, never) = 150;
+	private static var WALL_START_Y(default, never) = 200;
+
+	private static var LAVA_COUNT(default, never) = 10;
+	private static var LAVA_START_X(default, never) = 150;
+	private static var LAVA_START_Y(default, never) = 232;
+
+	private static var FIREBALL_COUNT(default, never) = 10;
```

Comment:
The world is so big now, you might want to up the fireball count to make it a bit more likely that the player has to dodge them.